src/components/RegisterPage/RegisterForm.tsx

```diff
@@ -0,0 +1,254 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import ReCAPTCHA from 'react-google-recaptcha';
+import {
+  validateRegisterPassword,
+  validateEmailField,
+  validateConsentsField,
+  validateCaptcha
+} from 'util/validators';
+import { selectPublisherConfig } from 'redux/publisherConfigSlice';
+import { fetchSettings, selectSettings } from 'redux/settingsSlice';
+import { selectPublisherConsents } from 'redux/publisherConsentsSlice';
+import { useAppDispatch, useAppSelector } from 'redux/store';
+import registerCustomer from 'api/Auth/registerCustomer';
+import getCustomerLocales from 'api/Customer/getCustomerLocales';
+import submitConsents from 'api/Customer/submitConsents';
+import Auth from 'services/auth';
+import Loader from 'components/Loader';
+import Consent from 'components/Consents';
+import { FromStyled, FormErrorStyled } from 'components/LoginPage/LoginStyled';
+import Button from 'components/Button';
+import EmailInput from 'components/EmailInput';
+import PasswordInput from 'components/PasswordInput';
+import { Consent as ConsentType } from 'types/Consents.types';
+import { RegisterFormProps } from './RegisterForm.types';
+import { RecaptchaError, RecaptchaWrapper } from './RegisterFormStyled';
+
+type Errors = {
+  email: string;
+  password: string;
+  consents: string;
+  captcha: string;
+};
+
+const errorsInitialState = {
+  email: '',
+  password: '',
+  consents: '',
+  captcha: ''
+};
+
+export const Register = ({ onSuccess }: RegisterFormProps) => {
+  const [email, setEmail] = useState<string>('');
```

Comment:
I think when we use not empty default value of state like string here or boolean in other lines, we don't have to set type in `<>` brackets because TS will infer it